src/plugins/intel_gna/src/backend/gna_limitations.cpp

```diff
@@ -999,9 +1047,13 @@ void Limitations::check_all_ops_supported(const std::shared_ptr<ov::Model>& mode
     std::stringstream error;
     // Walk through the transformed model
     for (auto& op : model->get_ops()) {
-        if (!is_op_supported(op, gna_precision, true)) {
-            error << "The plugin does not support layer " << op->get_friendly_name() << " (type " << op->get_type_name()
-                  << ")!" << std::endl;
+        try {
+            if (!is_op_supported(op, gna_precision, true)) {
+                error << "The plugin does not support layer " << op->get_friendly_name() << " (type "
+                      << op->get_type_name() << ")!" << std::endl;
+            }
+        } catch (const InferenceEngine::GeneralError& e) {
+            error << e.what() << std::endl;
```

Comment:
why do we want to catch this exception? 
do you want to change exception type?

Reply:
No, I want to catch all exceptions and list all of them later